topo/node/ceos/ceos.go

```diff
@@ -49,9 +52,16 @@ type Node struct {
 
 // Add validations for interfaces the node provides
 var (
-	_ node.Certer       = (*Node)(nil)
 	_ node.ConfigPusher = (*Node)(nil)
 	_ node.Resetter     = (*Node)(nil)
+
+	ethIntfRe  = regexp.MustCompile(`^Ethernet\d+(?:/\d+)?(?:/\d+)?$`)
+	mgmtIntfRe = regexp.MustCompile(`^Management\d+(?:/\d+)?$`)
+)
+
+const (
+	API      = "ceoslab.arista.com/v1alpha1"
```

Comment:
these constants should not be CAPS

this also has the side effect of exporting them

all lower case is preferred here

Reply:
+1